packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommitforckeditor5package.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const parserOptions = require( './parser-options' );
+const utils = require( './transform-commit-utils' );
+
+/**
+ * Parses a single commit:
+ *   - displays a log when the commit has invalid format of the message,
+ *   - filters out the commit if it should not be visible in the changelog,
+ *   - makes links to issues and user's profiles on GitHub.
+ *
+ * @param {Commit} commit
+ * @param {Boolean} [displayLog=true] Whether to display the logs.
+ * @returns {Commit}
+ */
+module.exports = function transformCommitForCkeditor5Package( commit, displayLog = true ) {
```

Comment:
OK, I wanted to be sure.

Then, the second question is whether we expect to have more functions related to "CKE 5 packages" and "CKE 5 dev packages"? If so, then perhaps we should move this code somewhere else. Keep it closer to each other. E.g. the tasks available in `index.js` are also CKE5 related only so soon we'll have "ForCKEditor5Package" everywhere. Hence, I wanted to propose to try to better organise this code. But I don't know it too well, so nothing concrete.